docs/user-guide/command-line-interface.md

```diff
@@ -69,7 +69,9 @@ Miscellaneous:
   --debug                    Output debugging information
   -h, --help                 Show help
   -v, --version              Outputs the version number
-  ```
+  --no-inline-config         Prevent comments from changing eslint rules -
```

Comment:
Actually, `inlineConfig` is `true` by default, but I think specifying it as `--no-inline-config` is the nicer API (like `--no-color`, above).